packages/plugins/plugin-assistant/package.json

```diff
@@ -121,6 +121,7 @@
     "use-deep-compare-effect": "^1.8.1"
   },
   "devDependencies": {
+    "@dxos/assistant-testing": "workspace:*",
```

Comment:
why? it's already in "dependencies"

Reply:
I think we could to with a todo somewhere to stop importing blueprints from assistant testing though